bench/gl-stats.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<script>
+let now = performance.now();
+window.performance.now = () => now;
```

Comment:
nit: Could you add a comment here indicating why `performance.now` is being stubbed and incremented by `16` on each render.